tests/yast2_gui/yast2_network_settings.pm

```diff
@@ -17,66 +17,80 @@
 use base "y2x11test";
 use strict;
 use testapi;
+use version_utils 'is_leap';
 
 sub run {
     my $self = shift;
-    $self->launch_yast2_module_x11('lan', match_args => [qw(yast2-lan-ui yast2_still_susefirewall2)], match_timeout => 60);
-    if (match_has_tag('yast2_still_susefirewall2')) {
+
+    # keyboard shorcuts
+    $cmd{global_options_tab} = 'alt-g';
+    $cmd{dhcp_client_id}     = 'alt-i';
+    $cmd{overview_tab}       = 'alt-v';
+    $cmd{add_device}         = 'alt-a';
+    $cmd{dynamic_address}    = 'alt-y';
+    $cmd{hostname_dns_tab}   = 'alt-s';
+    $cmd{routing_tab}        = 'alt-u';
+
+    $self->launch_yast2_module_x11('lan', target_match => [qw(yast2-lan-ui yast2_still_susefirewall2 yast2-lan-warning-network-manager)], match_timeout => 60);
+    if (match_has_tag 'yast2_still_susefirewall2') {
         record_soft_failure "bsc#1059569";
         send_key 'alt-i';
         wait_still_screen;
     }
-
-    #	Global Options
-    send_key 'alt-g';
+    elsif (match_has_tag 'yast2-lan-warning-network-manager' || is_leap('>=15.1')) {
```

Comment:
I recommend to not use the leap version here but rather a function to determine if networkmanager is the default or wicked. Within this function you can simply state that openSUSE has network manager as default except for older leap versions. Leap should never have any "newer" feature which Tumbleweed does not have